pkg/controller/sync/controller.go

```diff
@@ -648,8 +659,10 @@ func updatePropagatedVersion(adapter federatedtypes.FederatedTypeAdapter, versio
 	return err
 }
 
-// newVersion initializes a new version resource for the given cluster versions and template and override
-func newVersion(clusterVersions map[string]string, templateMeta *metav1.ObjectMeta, targetKind, overrideVersion string) *fedv1a1.PropagatedVersion {
+// newVersion initializes a new propagated version resource for the given
```

Comment:
(No action required) In future try to avoid mixing functional and non-functional (formattting in this case) changes in a single commit.

Reply:
Sorry about that. I was in refactoring mode and cleaning things up so I made these changes along the way. Afterwards it's difficult to split these changes up within the same file.